RenderStateNotation/include/RenderStateNotationParserImpl.hpp

```diff
@@ -52,7 +52,7 @@ class RenderStateNotationParserImpl final : public ObjectBase<IRenderStateNotati
 
     virtual Bool DILIGENT_CALL_TYPE ParseFile(const Char* FilePath, IShaderSourceInputStreamFactory* pStreamFactory) override final;
 
-    virtual Bool DILIGENT_CALL_TYPE ParseString(const Char* Source, Uint32 Length, IShaderSourceInputStreamFactory* pStreamFactory) override final;
+    virtual Bool DILIGENT_CALL_TYPE ParseString(const Char* Source, size_t Length, IShaderSourceInputStreamFactory* pStreamFactory) override final;
```

Comment:
size_t is platform and bitness-dependent. 
It is preferable to use stable types in the API (such Uint32 or Uint64).
I don't think the size of the string will be more than 4GB, so Uint32 is OK.